doc/build-instructions/Build_Instructions_V13.md

```diff
@@ -0,0 +1,654 @@
+
+- [autoconf 2.6.9](https://formulae.brew.sh/formula/autoconf)
+- [bash 4.4.23](https://formulae.brew.sh/formula/bash)
+- [binutils 2.32](https://formulae.brew.sh/formula/binutils)
+- [freetype 2.9.1](https://formulae.brew.sh/formula/freetype)
+- [git 2.19.2](https://formulae.brew.sh/formula/git)
+- [gnu-tar 1.3](https://formulae.brew.sh/formula/gnu-tar)
+- [gnu-sed 4.5](https://formulae.brew.sh/formula/gnu-sed)
+- [nasm 2.13.03](https://formulae.brew.sh/formula/nasm)
+- [pkg-config 0.29.2](https://formulae.brew.sh/formula/pkg-config)
+- [wget 1.19.5 ](https://formulae.brew.sh/formula/wget)
+
+[Freemarker V2.3.8](https://sourceforge.net/projects/freemarker/files/freemarker/2.3.8/freemarker-2.3.8.tar.gz/download) is also required, which can be obtained and installed with the following commands:
+
+```
+cd /<my_home_dir>
+wget https://sourceforge.net/projects/freemarker/files/freemarker/2.3.8/freemarker-2.3.8.tar.gz/download -O freemarker.tgz
+tar -xzf freemarker.tgz freemarker-2.3.8/lib/freemarker.jar --strip-components=2
+rm -f freemarker.tgz
+```
+
+Bash version 4 is required by the `./get_source.sh` script that you will use in step 2, which is installed to `/usr/local/bin/bash`. To prevent problems during the build process, make Bash v4 your default shell by typing the following commands:
+
+```
+# Find the <CURRENT_SHELL> for <USERNAME>
+dscl . -read <USERNAME> UserShell
+
+# Change the shell to Bash version 4 for <USERNAME>
+dscl . -change <USERNAME> UserShell <CURRENT_SHELL> /usr/local/bin/bash
+
+# Verify that the shell has been changed
+dscl . -read <USERNAME> UserShell
+```
+
+Set the following environment variables:
+
+```
+export SED=/usr/local/bin/gsed
+export TAR=/usr/local/bin/gtar
+```
+
+### 2. Get the source
+:apple:
+First you need to clone the Extensions for OpenJDK for OpenJ9 project. This repository is a git mirror of OpenJDK without the HotSpot JVM, but with an **openj9** branch that contains a few necessary patches.
+
+Run the following command:
+```
+git clone https://github.com/ibmruntimes/openj9-openjdk-jdk13.git
+```
+Cloning this repository can take a while because OpenJDK is a large project! When the process is complete, change directory into the cloned repository:
+```
+cd openj9-openjdk-jdk13
+```
+Now fetch additional sources from the Eclipse OpenJ9 project and its clone of Eclipse OMR:
+
+```
+bash ./get_source.sh
+```
+
+:pencil: **OpenSSL support:** If you want to build an OpenJDK with OpenJ9 binary with OpenSSL support and you do not have a built version of OpenSSL v1.0.2 or v1.1.x available locally, you must obtain a prebuilt OpenSSL binary.
+
+### 3. Configure
+:apple:
+When you have all the source files that you need, run the configure script, which detects how to build in the current build environment.
+
+```
+bash configure --with-freemarker-jar=/<my_home_dir>/freemarker.jar \
+               --with-boot-jdk=<path_to_macOS_JDK11> \
```

Comment:
jdk12 or 13

Reply:
Seems strange that some platforms say to use `--with-boot-jdk=` and some don't. Unless you have the boot-jdk on your path, it always needs to be specified.